tests/datalab/test_datalab.py

```diff
@@ -237,6 +237,10 @@ def test_find_issues_without_values_in_issue_types_raises_warning(self, lab, pre
             in warning_message
         )
 
+    def test_find_issues_with_low_memory(self, lab, pred_probs):
+        issue_types = {"label": {"clean_learning_kwargs": {"low_memory": True}}}
+        lab.find_issues(pred_probs=pred_probs, issue_types=issue_types)
```

Comment:
IMO this block should instead be added at the end of an existing unit test that does the find_issues without low_memory, and then you should finally check the jaccard similarity between the 2 runs.